site/src/pages/ManagementSettingsPage/CustomRolesPage/CustomRolesPageView.tsx

```diff
@@ -29,12 +29,12 @@ import { Link as RouterLink, useNavigate } from "react-router-dom";
 import { docs } from "utils/docs";
 import { PermissionPillsList } from "./PermissionPillsList";
 
-export type CustomRolesPageViewProps = {
+interface CustomRolesPageViewProps {
```

Comment:
My goal was just to be consistent and I saw more interfaces used for all the organization related pages. I don't personally have a strong preference other than being consistent.

Reply:
I like `interface` because I think `interface WibbleProps extends WobbleProps {}` is a lot clearer than `type WibbleProps = WobbleProps & {}` in the places where we need it. and I think we just generally use it in more places rn.